deepspeed/runtime/zero/partitioned_param_coordinator.py

```diff
@@ -56,14 +56,6 @@ def __setitem__(self, param: Parameter, handle: AllGatherCoalescedHandle) -> Non
 
 
 class PartitionedParameterCoordinator:
-    FORWARD_FETCH_SUBMIT = 'forward_fetch_submit'
```

Comment:
Please don't remove profiling code. Instead, please ensure that it is disabled by default.

Reply:
Fixed

Reply:
The prefetch and backward profilers (e.g, `FORWARD_PREFETCH_SUBMIT` and `BACKWARD_FETCH_SUBMIT`) are not yet restored.